bigchaindb/core.py

```diff
@@ -633,20 +520,9 @@ def deserialize(data):
 
         Returns:
             dict: dict resulting from the serialization of a JSON formatted string.
-
         """
-        return json.loads(data, encoding="utf-8")
-
-    @staticmethod
-    def timestamp():
-        """Static method to calculate a UTC timestamp with microsecond precision.
-
-        Returns:
-            str: UTC timestamp.
 
-        """
-        dt = datetime.utcnow()
-        return "{0:.6f}".format(time.mktime(dt.timetuple()) + dt.microsecond / 1e6)
+        return json.loads(data, encoding="utf-8")
 
     @staticmethod
     def generate_keys():
```

Comment:
I left it there because some tests rely on that method, but I can remove it (and fix the tests).